### PR TITLE
♻️ Frontend: use PATCH instead of PUT for ``/projects`` I

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR refactors the dashboard not to ``PUT`` the ``projects`` anymore.

 ``projects``:``patch``:
- [x] name
- [x] description
- [x] thumbnail
- [x] accessRights
- [x] quality
- [x] classifiers
- [x] dev

 ``projects/node``:``patch``:
- [x] version
- [x] bootOptions

More redesign is required to use the ``PATCH`` from the study editor, but it will come in a follow up PR.

#### Bonus:
- [x] Fix: bring back the footer layout (hits and TSR) to the service card

## Related issue/s

follows https://github.com/ITISFoundation/osparc-simcore/pull/5802
closes https://github.com/ITISFoundation/osparc-simcore/issues/5617

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
Looking forward :tada:

File: services/static-webserver/client/source/class/osparc/dashboard/CardBase.js [hunk 1]
```diff
@@ -299,23 +297,23 @@ qx.Class.define("osparc.dashboard.CardBase", {
...
+          uuid = resourceData.uuid ? resourceData.uuid : null;
+          owner = resourceData.prjOwner ? resourceData.prjOwner : "";
+          workbench = resourceData.workbench ? resourceData.workbench : {};
           break;
         case "template":
-          uuid = resourceData.uuid ? resourceData.uuid : uuid;
-          owner = resourceData.prjOwner ? resourceData.prjOwner : owner;
-          workbench = resourceData.workbench ? resourceData.workbench : workbench;
+          uuid = resourceData.uuid ? resourceData.uuid : null;
+          owner = resourceData.prjOwner ? resourceData.prjOwner : "";
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/dashboard/CardBase.js:311]: TIP: i would have a mechanism that loads all these defaults from the models in services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/dashboard/CardBase.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/dashboard/CardBase.js:311]: 👌

**Review by GitHK — APPROVED** [review]
👍

**Review by ignapas — CHANGES_REQUESTED** [review]
just a minor

File: services/static-webserver/client/source/class/osparc/info/StudyUtils.js [hunk 2]
```diff
@@ -391,5 +390,48 @@ qx.Class.define("osparc.info.StudyUtils", {
...
+            // A bit hacky, but it's not sent back to the backend
+            studyData["lastChangeDate"] = new Date().toISOString();
+            resolve();
+          })
+          .catch(err => reject(err));
+      });
+    },
+
+    patchStudyData: function(studyData, fieldKey, value) {
+      return new Promise((resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/info/StudyUtils.js]: Awesome, thanks!

File: services/static-webserver/client/source/class/osparc/dashboard/CardBase.js (hunk 1 above)
Comment (sanderegg) [reply services/static-webserver/client/source/class/osparc/dashboard/CardBase.js:311]: hmm, can I get in the loop please? I am not sure I get the point of having delete rights on the services.

**Review by sanderegg — APPROVED** [review]
very cool thanks!

File: services/static-webserver/client/source/class/osparc/info/ServiceUtils.js [hunk 3]
```diff
@@ -295,23 +295,19 @@ qx.Class.define("osparc.info.ServiceUtils", {
...
+      const patchData = {};
+      patchData[fieldKey] = value;
+      const params = {
+        url: osparc.data.Resources.getServiceUrl(
+          serviceData["key"],
+          serviceData["version"]
+        ),
+        data: patchData
+      };
+      osparc.data.Resources.fetch("services", "patch", params)
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/info/ServiceUtils.js:307]:
```suggestion
      return osparc.data.Resources.fetch("services", "patch", params)
```

Comment (ignapas) [reply services/static-webserver/client/source/class/osparc/info/ServiceUtils.js:307]: like that you can also
``
ServiceUtils.patchServiceData(...).then(...)...
``

Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/info/ServiceUtils.js]: Forgot to return it, thanks

**Review by ignapas — APPROVED** [review]
lgtm

**Pull request merged**